daprdocs/content/en/js-sdk-docs/js-actors/_index.md

```diff
@@ -64,6 +64,36 @@ export default class ParkingSensorImpl extends AbstractActor implements ParkingS
 }
 ```
 
+### Configuring Actor Runtime
+
+To configure actor runtime, use the `DaprClientOptions`. The various parameters and their default values are documented at [How-to: Use virtual actors in Dapr](https://docs.dapr.io/developing-applications/building-blocks/actors/howto-actors/#configuration-parameters).
+
```

Comment:
Though we have mentioned in `ActorRuntimeOptions` that interval/timeout options use https://pkg.go.dev/time#ParseDuration format, can we specify explicitly the format to be used by duration kind of things? It may be handy for the user.

Reply:
Yeah, it maybe helpful, just wondering if it will clutter the comment too much, and then if something changes in the `time.ParseDuration` our comment will be stale. WDYT?

Reply:
I think it would be definitely helpful. As an example, when I was very new in Dapr, I was trying to set some duration property in a yaml and I had to search at some places/fail few times due to that property before I could determine the correct syntax. For any new user, it should be right there wherever we are documenting. 
I don't think `time.ParseDuration` syntax would change as such. In general, a better way for any duration filed can be if as Dapr are able to define the syntax that we recommend at a single place and any/all duration document can refer that.

Reply:
> In general, a better way for any duration filed can be if as Dapr are able to define the syntax that we recommend at a single place and any/all duration document can refer that.

Agreed. This should be locally updated in runtime documentation, e.g., https://docs.dapr.io/developing-applications/building-blocks/actors/howto-actors/#actor-timers-and-reminders, and we can refer that.

Reply:
https://github.com/dapr/docs/issues/2766 created to track.

Reply:
Can we at least for now provide link to https://pkg.go.dev/time#ParseDuration ?

Reply:
Oh yeah, sure! I was thinking about this src/types/actors/ActorRuntimeOptions.ts all this while, which has the link to it. Let me add it here, good catch!

Reply:
Sure, it's present in src/types/actors/ActorRuntimeOptions.ts (for developers), adding it here.